### ISSUE TITLE
How can I define the behavior of a customized log level?

### ISSUE THREAD

**Issue by ylorn** [CocoaLumberjack/CocoaLumberjack#671]
I integrated CocoaLumberjack into my project. I have 4 log levels, and they are `Error`(0001), `Warning`(0010), `Info`(0100) and `Verbose`(1000). The global log level is set to `Verbose`, and I added `DDTTYLogger` and a instance of `DDFileLogger`to the `DDLog`.

So now, all logs of all levels will be printed to the console and written to a local file, which is what I expected. 

Now, I want to add 2 more customized log level, say `logLevelPrintToConsole` and `logLevelSaveToFile`. And all other settings remain the same, which means the global log level is still `Verbose`, and `Error`, `Warning`, `Info` and `Verbose` will still be printed and saved. 
- I want any log message is handled by `logLevelPrintToConsole` to be **only printed to the console**, **but _not_ saved locally**.
- I want any log message is handled by `logLevelSaveToFile` to be **only saved to the local file, but _not_ displayed on the console**.

To sum it up, what I want to achieve is:
- The `DDTTYLogger`should handle:
  - `Error`, `Warning`, `Info`, `Verbose`
  - and `logLevelPrintToConsole`
- The instance of `DDFileLogger`should handle:
  - `Error`, `Warning`, `Info`, `Verbose`
  - and `logLevelSaveToFile`

How can achieve such goal?

Thanks in advance!

**Comment by ylorn** [comment]
I was wondering if the following solution works:


 | Flag | Level
------------ | ------------ | -------------
Error | 000001 | 000001
Warning | 000010 | 000011
Info | 000100 | 000111
Verbose | 001000 | 001111
logLevelPrintToConsole | 010000 | 011111
logLevelSaveToFile | 100000 | 101111

So if I do `[DDLog addLogger:[DDTTYLogger sharedInstance] withLevel: logLevelPrintToConsole];`, log messages with level `logLevelSaveToFile` will *not* be printed to the console.

And if I do 

```
DDFileLogger *fileLogger = [[DDFileLogger alloc] init];
[DDLog addLogger:fileLogger withLevel: logLevelSaveToFile];
```
then log messages with level `logLevelPrintToConsole ` will not be saved to the local file.


Is this the correct way to do it?

**Comment by rivera-ernesto** [comment]
In any case you would need to subclass the loggers to ignore console/file-only messages but I think it would be cleaner to use `tag`s instead of levels, because that what they are: you send a message of a certain level and just tag it to be ignored/or not by a logger.

**Comment by ylorn** [comment]
@rivera-ernesto I followed [Custom Log Levels](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/Documentation/CustomLogLevels.md) to create my own MyLog.h to define my customized log levels to achieve what I want to.

It works.

But I think want you suggest will make my code more readable, I'll definitely give it a try.

Thanks.

**Comment by rivera-ernesto** [comment]
More readable and allow you to still apply different log leves to your tagged messages.
